fetch.bs

```diff
@@ -4658,6 +4660,35 @@ steps. They return a <a for=/>response</a>.
    <var>fetchParams</var>.
   </ol>
 
+ <li>
+  <p>If the following are true
+
+  <ul class=brief>
+   <li><p><var>response</var>'s <a for=response>status</a> is 421
```

Comment:
Please place a newline below.

Please see https://github.com/whatwg/fetch#formatting for the style.
Ditto below.